### PR TITLE
feat/docs/fix :sparkles: Added momentum based gradient descent to increase the accuracy

### PR DESCRIPTION
**Pull request by ayushi2626**
Used momentum-based gradient descent and it increased the accuracy. I have attached the screenshots below:

Accuracy_ss

SampleData_ss

### CODE REVIEW

**Comment by UppuluriKalyani** [comment]
@ayushi2626 hey even though the accuracy is 90 above but the prediction isn't well..it's wrongly classifying please check once

**Comment by ayushi2626** [comment]
Hey are you talking about the predictions using the images one before saving the model and one after saving the model?

**Comment by ayushi2626** [comment]
Sure, let me check it once

**Comment by ayushi2626** [comment]
Hey I just pushed into the same issue here. So sorry I am new to open-sourcing so please do let me know if there is anything more to do done. Thank you so much!

**Comment by UppuluriKalyani** [comment]
@ayushi2626 it's ok don't worry, it's predicting good afternoon saving the model but not on training data...ok you did it good.

**Comment by ayushi2626** [comment]
Thank you so much!
